src/binder/bind/bind_comment_on.cpp

```diff
@@ -9,11 +10,8 @@ std::unique_ptr<BoundStatement> Binder::bindCommentOn(const parser::Statement& s
     auto& commentOnStatement = reinterpret_cast<const parser::CommentOn&>(statement);
     auto tableName = commentOnStatement.getTable();
     auto comment = commentOnStatement.getComment();
-
     validateTableExist(tableName);
-    auto catalogContent = catalog.getReadOnlyVersion();
-    auto tableID = catalogContent->getTableID(tableName);
-
+    auto tableID = catalog.getTableID(clientContext->getTx(), tableName);
```

Comment:
Should we use trx or tx as the short hand for transaction?
I see other places use trx instead of tx

Reply:
I think we should move to `tx`. This term is more common in other systems.